homeassistant/components/supla/__init__.py

```diff
@@ -92,7 +93,8 @@ async def async_setup(hass: HomeAssistant, base_config: ConfigType) -> bool:
 
 
 async def discover_devices(hass, hass_config):
-    """Run periodically to discover new devices.
+    """
+    Run periodically to discover new devices.
```

Comment:
Please revert this change.

Reply:
This must have been introduced by Black. Reverted.